packages/flutter_test/lib/src/binding.dart

```diff
@@ -189,6 +189,32 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     debugDisableShadows = disableShadows;
   }
 
+  /// Deprecated. Will be removed in a future version of flutter.
```

Comment:
nit: I think we capitalize our brand:

```suggestion
  /// Deprecated. Will be removed in a future version of Flutter.
```

Reply:
Good catch! Updated!